helmfile/values/elasticsearch-backup.yaml.gotmpl

```diff
@@ -8,7 +8,7 @@ resources:
     cpu: 10m
     memory: 32Mi
 
-snapshotRepository: {{ .Values.elasticsearch.snapshotRepository }}
+snapshotRepository: s3_{{ .Values.global.cloudProvider }}_7.x
```

Comment:
This increases dependency on the `cloudProvider` value.
I would suggest keeping the `snapshotRepository` value and setting the value in `init` depending on `cloudProvider` instead.

Reply:
Sure, I can do it that way instead.

Reply:
Decided to update the opendistro chart as well to align them. See this PR https://github.com/elastisys/opendistro-build/pull/30